src/components/GoogleButton/GoogleButton.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import styles from './GoogleButton.module.css'
+import googleLogo from '../../assets/icons/googleLogo.png'
+import googleLogo2x from '../../assets/icons/googleLogo@2x.png'
+import googleLogo3x from '../../assets/icons/googleLogo@3x.png'
+// import url from '../../api/apiEndpoint';
+
+const GoogleButton = () => {
+  const handleClick = () => {
+    // document.location.replace(url.googleAuth);
+    console.log('go to auth by google')
+  }
+
+  return (
+    <button type='button' className={styles.button} onClick={handleClick}>
```

Comment:
it's tag a not button. `<a href="https://domain.com/api/v1/auth/google">Google</a>`

Reply:
Done